MyHashSet.java

```diff
@@ -52,4 +52,8 @@ public DLList<E> toDLList(){
         return hashList;
     }
 
+    public String toString()
+    {
+        return toDLList().toString();
+    }
```

Comment:
this is a really weird one, in that i had to add it because of some weird code in `Screen.java`

basically when we initially display all the values of myhashmap, we use =
https://github.com/zwlwilliamzha/HashMap-Lab/blob/166875adcafe4d71da0338dc4b660937adca7858/Screen.java#L149

but then after we delete one entry, we use
https://github.com/zwlwilliamzha/HashMap-Lab/blob/166875adcafe4d71da0338dc4b660937adca7858/Screen.java#L395

Reply:
Here is what the user interface looks like without this change
I deleted CA, then it basically printed the hashmap memory location

<img width="779" alt="Screenshot 2023-11-09 at 10 40 45 PM" src="https://github.com/zwlwilliamzha/HashMap-Lab/assets/39246862/a2180008-45c4-4bf7-97e8-e35c00800898">